docs/guide/usage.md

```diff
@@ -0,0 +1,86 @@
+## Create a Store with Rodux
+Create your store as normal with [Rodux](https://github.com/Roblox/Rodux):
+
+```lua
+local store = Rodux.Store.new(function(state, action)
+	state = state or {
+		value = 0,
+	}
+
+	if action.type == "increment" then
+		return {
+			value = state.value + 1,
+		}
+	end
+
+	return state
+end)
```

Comment:
Wonder if its perhaps a little more readable to separately define the reducer above instead of inlining it.  Might just help to make it look like what people are used to seeing when they see `Rodux.Store.new(...`

Reply:
Woops, this was a very old example that I copy-pasted to start working on.